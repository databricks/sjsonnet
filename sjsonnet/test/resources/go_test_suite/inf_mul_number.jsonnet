1e300 * 1000000000