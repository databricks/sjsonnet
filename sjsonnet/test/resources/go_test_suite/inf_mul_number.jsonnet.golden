sjsonnet.Error: overflow
    at [BinaryOp *].(inf_mul_number.jsonnet:1:7)

